scripts/core/config.ts

```diff
@@ -3,8 +3,12 @@ import {appConfig} from 'appConfig';
 import {IArticle} from 'superdesk-api';
 
 export function isMediaEditable(item?: IArticle) {
-    if (item !== undefined && item._fetchable !== undefined) {
-        return item._fetchable;
+    if (item != null) {
+        if (item._type != null) {
+            return item._type !== 'externalsource';
+        } else if (item._fetchable != null) {
+            return item._fetchable;
```

Comment:
shouldn't it be more like:
```ts
if (item != null) {
  if (item._fetchable != null) {
    return item._fetchable;
  } else if (item._type !== 'externalsource') {
    return true;
  }
}
...
```

Reply:
because I think in general external items will have `_type` set so it wouldn't check `_fetchable` flag